tests/Unit/ChangelogTest.php

```diff
@@ -172,9 +172,8 @@ public static function provideChangedlogFiles(): iterable
             ->in(\dirname(__DIR__, 2) . '/src')
             ->name('CHANGELOG.md');
 
-        /** @var \SplFileInfo $file */
```

Comment:
`Finder` defines its iterable type as the `Symfony\Component\Finder\SplFileInfo`, which is the class defining `getRelativePath` (as it is relative to the folder in which the finder is searching).
This `@var` was probably added at a time when Symfony did not have the phpdoc annotation providing the info.